src/routes/models/[id]/+page.svelte

```diff
@@ -0,0 +1,578 @@
+              return goalSpecA.priority - goalSpecB.priority;
+            }
+            return 0;
+          });
+
+        const prevPriority = selectedGoalModelSpecifications[id].priority ?? 0;
+        const nextPriority = priority != null ? Math.min(priority, goalModelSpecificationsList.length) : 0;
+        const priorityModifier = nextPriority < prevPriority ? 1 : -1;
+
+        selectedGoalModelSpecifications = goalModelSpecificationsList
+          .map(goalSpecification => {
+            if (
+              goalSpecification.selected &&
+              goalSpecification.priority != null &&
+              ((priorityModifier < 0 &&
+                goalSpecification.priority >= prevPriority &&
+                goalSpecification.priority <= nextPriority) ||
+                (goalSpecification.priority <= prevPriority && goalSpecification.priority >= nextPriority))
+            ) {
+              return {
+                ...goalSpecification,
+                priority: goalSpecification.priority + priorityModifier,
+              };
+            }
+
+            return goalSpecification;
+          })
+          .reduce(
+            (prevGoalSpecificationMap: Record<number, AssociationSpecification>, goalModelSpecification) => {
+              return {
+                ...prevGoalSpecificationMap,
+                [goalModelSpecification.id]: {
+                  priority: goalModelSpecification.priority,
+                  revision: goalModelSpecification.revision,
+                  selected: goalModelSpecification.selected,
+                },
+              };
+            },
+            {
+              [id]: {
+                priority: nextPriority,
+                revision,
+                selected,
+              },
+            },
+          );
+        break;
+      }
+      case 'constraint':
+      default:
+        selectedConstraintModelSpecifications = {
+          ...selectedConstraintModelSpecifications,
+          [id]: {
+            revision,
+            selected,
+          },
+        };
+    }
+  }
+
+  function onViewMetadata(event: CustomEvent<number>) {
+    const { detail: id } = event;
+
+    switch (selectedAssociation) {
+      case 'condition':
+        window.open(`${base}/scheduling/conditions/edit/${id}?${SearchParameters.MODEL_ID}=${$model?.id}`);
+        break;
+      case 'goal':
+        window.open(`${base}/scheduling/goals/edit/${id}?${SearchParameters.MODEL_ID}=${$model?.id}`);
+        break;
+      case 'constraint':
+      default:
+        window.open(`${base}/constraints/edit/${id}?${SearchParameters.MODEL_ID}=${$model?.id}`);
+    }
+  }
+</script>
+
+<PageTitle subTitle={data.initialModel.name} title="Models" />
+
+<CssGrid columns="0.25fr 3px 1fr">
+  <Panel>
+    <svelte:fragment slot="header">
+      <SectionTitle>Model info</SectionTitle>
+    </svelte:fragment>
+
+    <svelte:fragment slot="body">
+      <ModelForm
```

Comment:
Consider using the 16px padding that the models page uses so that the model info form 
<img width="299" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/70a1b73b-aec1-4939-abef-5b27628bb1b4">
matches the other pages
<img width="273" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/73c7c8f1-6cb4-4772-b776-89aec48a68dc">